near-sdk/src/store/unordered_set/mod.rs

```diff
@@ -0,0 +1,798 @@
+mod impls;
+mod iter;
+
+use super::{FreeList, LookupMap, ERR_INCONSISTENT_STATE};
+use crate::crypto_hash::{CryptoHasher, Sha256};
+use crate::store::free_list::FreeListIndex;
+use crate::store::unordered_set::iter::{
+    Difference, Drain, Intersection, Iter, SymmetricDifference, Union,
+};
+use crate::{env, IntoStorageKey};
+use borsh::{BorshDeserialize, BorshSerialize};
+use std::borrow::Borrow;
+use std::fmt;
+
+pub struct UnorderedSet<T, H = Sha256>
```

Comment:
Can you add some docs for this? Can just pull from `std::collections::HashSet` and modify, like it seems you've done with others :)

Reply:
Oops, missed the most important doc of them all :)